antigen-hs.cabal

```diff
@@ -0,0 +1,22 @@
+name:                antigen-hs
+version:             0.1.0.0
+synopsis:            A fast zsh plugin manager
+description:         Please see README.md
+homepage:            https://github.com/Tarrasch/antigen-hs#readme
+license:             MIT
+license-file:        LICENSE.txt
+author:              Arash Rouhani
+maintainer:          arash@spotify.com
+copyright:           2014 Arash Rouhani
+category:            Tools
+build-type:          Simple
+cabal-version:       >=1.10
+
+library
+  exposed-modules:     Antigen
+  default-language:    Haskell2010
+  build-depends:       base >= 4.7 && < 5
```

Comment:
Do you really need to be this strict @Pitometsu?

It's ridiculous if 4.6 doesn't work as that what's shipped with Ubuntu 14.04 LTS (as it uses 7.6.3).